Python/import.c

```diff
@@ -1745,7 +1745,7 @@ PyObject *
 PyImport_Import(PyObject *module_name)
 {
     PyThreadState *tstate = _PyThreadState_GET();
-    static PyObject *silly_list = NULL;
+    PyObject *from_list = NULL;
     static PyObject *builtins_str = NULL;
     static PyObject *import_str = NULL;
```

Comment:
static variables are bad, they don't work in subinterpreters. You can use the _Py_IDENTIFIER() API with _PyUnicode_FromId() (return a borrowed reference), it is compatible with subinterpreters.